src/dash/models/DashManifestModel.js

```diff
@@ -41,6 +41,12 @@ import ObjectUtils from '../../streaming/utils/ObjectUtils';
 import URLUtils from '../../streaming/utils/URLUtils';
 import FactoryMaker from '../../core/FactoryMaker';
 
+Number.isInteger = Number.isInteger || function (value) {
```

Comment:
Hi @bbcrddave ,

if the function Number.isInteger is undefined in a specific browser, define a polyfill function could be useful in a another part of javascript code. In this case, it could be cleaner to put this polyfill function in a specific file (in utils folder).
On another side, this function is currently only use in DashManifestModel. So, your option could indeed be selected.
In any case, if Number.IsInteger is already defined, we don't override it.

Nicolas